.travis.yml

```diff
@@ -3,7 +3,7 @@ env:
  global:
    - DONT_FETCH_GAP_JULIA_PKG=yes
    - JULIAROOT=$TRAVIS_BUILD_DIR/julia
-   - GAPROOT=$TRAVIS_BUILD_DIR/gap
+   - GAP_ROOT=$TRAVIS_BUILD_DIR/gap
```

Comment:
Well, this was basically done because I introduced the `GAP_ROOT` in the build.jl, and then the cascade of changes piled up. I will change them all to `GAPROOT`